### DISCUSSION TITLE
NocoDB with Obsidian or Logseq or Dendron [Jun 08]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2386] [Discord Threads]
> 🙋‍♂️ **Hioko**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### please expand you usecase  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/983799590354563123)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
